curvefs/src/metaserver/trash.h

```diff
@@ -52,6 +52,19 @@ using ::curve::common::InterruptibleSleeper;
 using ::curvefs::client::rpcclient::MdsClient;
 using ::curvefs::client::rpcclient::MdsClientImpl;
 
+struct TrashItem {
```

Comment:
The `TrashItem` is not need. Map<inodeId, dtime> is enough and better to find process when delete.